### PR TITLE
Add style guide item on capitalization

### LINKED ISSUE #665: Style: Capitalization of variable names in specifications
The style guide should have an entry regarding the capitalization of variable names in specifications.

Today it seems most common that every word is capitalized (e.g. "Domain Part"), but sometimes only the first word is capitalized (e.g. "Mail address"). I think we should be consistent on this point.

### PR DESCRIPTION
**Pull request by mattias-p**
Fixes #665.

### CODE REVIEW

**Comment by matsduf** [comment]
Is this for v2020.1?

**Comment by mattias-p** [comment]
Why not 2019.2?

**Comment by matsduf** [comment]
> Why not 2019.2?

I looked at the issue it solves (#65) and that is tagged with the milestone v2020.1. So that is why I asked as I did.

File: docs/internal-documentation/maintenance/StyleGuide.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ The following style guide applies to document text and certain file names specif
 
 * Zonemaster is always written as **Zonemaster**, never shortened (e.g. "ZM") or uncapitalized (e.g. "zonemaster"). 
 * Case-insensitive regexes are always written out with the ignore-case flag (e.g. `/[a-z]/i`).
+* For variable names in test case specifications each word is capitalized (e.g. "Domain Part").
```
Comment (matsduf) [inline docs/internal-documentation/maintenance/StyleGuide.md]:
```
If a word is an abbreviation, it is usually written with all capitals, e.g. "DNS".
```

Comment (mattias-p) [reply docs/internal-documentation/maintenance/StyleGuide.md]: I'll add that.

**Pull request merged**